libqtile/widget/volume.py

```diff
@@ -72,13 +68,17 @@ class Volume(base._TextBox):
     ]
 
     def __init__(self, **config):
-        base._TextBox.__init__(self, '0', width=bar.CALCULATED, **config)
+        base._TextBox.__init__(self, '      ', width=bar.CALCULATED, **config)
```

Comment:
Why this? It actually may be better to have some error message as the default string.